File: SampleSetup/Protocol/Body/AbstractReturn.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body
{
    public abstract class AbstractReturn
    {
        public DateTime InsertDate = DateTime.UtcNow;
        
        public int ResultEnumId = 0;

        public string ResultDescription = "";
    }
}

File: SampleSetup/Output/PHP_8/Body/AbstractReturn.php
<?php
namespace SampleProject\Protocol\Body;

class AbstractReturn {
    public string $InsertDate;

    public int $ResultEnumId;

    public string $ResultDescription;

    public function __construct()
    {
        $this->InsertDate = "";
        $this->ResultEnumId = 0;
        $this->ResultDescription = "";
    }
}
